common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1870,19 +1870,19 @@ def endOutput(self, header=[], padChar='-', trimOwner=False, trimHeader=False):
 		# binary	- marshalled python
 		# text		- default (for humans)
 
-		format = self._params.get('output-format')
-		if not format:
-			format = 'text'
+		output_format = self._params.get('output-format')
+		if not output_format:
+			output_format = 'text'
 
-		tokens = format.split(':')
+		tokens = output_format.split(':')
```

Comment:
Not your code, but I'd probably change this `token` parsing (lines 1877-1883) to be:
```
format_args = None
if ':' in output_format:
    output_format, format_args, _ = output_format.split(':', 2)
```